src/main/kotlin/app/revanced/cli/command/MainCommand.kt

```diff
@@ -71,6 +71,26 @@ internal object MainCommand : Runnable {
             return
         }
 
+        if (includedPatches.isEmpty()) {
+            for (patchBundlePath in patchBundles) for (patch in JarPatchBundle(patchBundlePath).loadPatches()) {
+                if (
+                //get key from console
+                    System.console().readLine("\"--include ${patch.patchName}\" ? Y/n: ").let {
```

Comment:
Make sure to try-catch this, it might fail sometimes. You can either exit if it happens, or continue.